### ISSUE TITLE
composer is not getting silex2 version

### ISSUE THREAD

**Issue by quantegy** [danadesrosiers/silex-annotation-provider#20]
I have my minimum-stability set to dev, but composer update barfs on your composer install instructions

```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package ddesrosiers/silex-annotation-provider ~2.0@dev exists as ddesrosiers/silex-annotation-provider[1.1.x-dev, dev-master, v0.1.0, v0.1.1, v0.1.2, v0.2.0, v0.2.1, v1.0.0, v1.1.0] but these are rejected by your constraint.
```

**Comment by danadesrosiers** [comment]
Well, that's not right.  The documentation is wrong.  I don't know how that happened.  Anyway, try `"ddesrosiers/silex-annotation-provider": "dev-master"`.  I'm working on getting a proper 2.0 release out.

**Comment by DerManoMann** [comment]
👍 for a version 2 release for Silex2

**Comment by marabesi** [comment]
Hello guys is there any update on that ?

**Comment by danadesrosiers** [comment]
I just published version 2.0.  Sorry for the delay.  I meant to do some cleanup first, but I haven't had the time and it can't wait any longer.
